libdnf/transaction/transaction_item_action.cpp

```diff
@@ -89,7 +89,7 @@ std::string transaction_item_action_to_letter(TransactionItemAction action) {
 }
 
 
-bool transaction_item_action_is_forward(TransactionItemAction action) {
+bool transaction_item_action_is_inbound(TransactionItemAction action) {
```

Comment:
@Conan-Kudo hey, cold you help out here? :grin: 

We're debating the naming. Some people in the team don't like the inbound / outbound naming. One suggested alternative was incoming / outgoing. IMO those come out mostly the same, except inbound / outbound seem marginally cleaner to me. We don't think either is great, but what do we non-native speakers know. Can you think of anything better?

Reply:
Probably more descriptive terms for this would be `additive` and `reductive`? What are you trying to describe?

Reply:
Oh, sorry, I somehow thought you'd know what I'm talking about. Have a look at the bodies of the two methods (`transaction_item_action_is_inbound()` and `transaction_item_action_is_inbound()`).

Say for an upgrade operation, there are actually two transaction items. One for the package being "installed", the action of which is UPGRADE. Another one for the package being "removed", the action of which is UPGRADED. The former is `inbound` in my current terminology, the latter `outbound`.

Note the previous naming (`forward` / `backward`) was just wrong, because you'd think DOWNGRADE is a `backward` action, but it was actually `forward` (now `inbound`, it's the new package being installed, even though lower version).

With that, I don't think `additive` and `reductive` are great?

Reply:
Yeah, I agree that `additive` and `reductive` probably don't make sense. I would say `action` and `effect` but you don't have an "effect" for INSTALLED or an "action" for REMOVE.